src/docs/12_glossary.adoc

```diff
@@ -23,9 +23,12 @@ Potentially more columns in case you need translations.
 
 ****
 
-[options="header"]
+[options="header",cols="1,4"]
 |===
 | Term         | Definition
-| <Term-1>     | <definition-1>
-| <Term-2>     | <definition-2>
+| *_SOLID_*     | Solid (derived from "social linked data") is a proposed set of conventions and tools for building decentralized social applications based on Linked Data principles. Solid is modular and extensible and it relies as much as possible on existing W3C standards and protocols.
+| *_POD_*     | User data is stored in its own Solid POD. PODs are like secure USB sticks for the Web, that you can access from anywhere. 
+| *_React_*     | React is a declarative, efficient, and flexible JavaScript library for building user interfaces. It lets you compose complex UIs from small and isolated pieces of code called “components”.
+| *_Asciidoctor_*     | <definition-1>
+| *_Node.js_*     | <definition-1>
```

Comment:
Are these definitions missing on purpose?? If not, fill them and merge, everything else is good!! :)